pjsip/src/pjsip-simple/presence_body.c

```diff
@@ -119,7 +119,7 @@ PJ_DEF(pj_status_t) pjsip_pres_create_pidf( pj_pool_t *pool,
 	  pj_parsed_time pt;
 
 	  pj_gettimeofday(&tv);
-	  /* TODO: convert time to GMT! (unsupported by pjlib) */
+	  pj_time_local_to_gmt(&tv);
```

Comment:
minor: probably add a log or a doc either here or in os_time implementation, that this can be unsupported?